### ISSUE TITLE
Error "Argument list too long" when running sanity checks

### ISSUE THREAD

**Issue by benediktahrens** [UniMath/UniMath#1819]
```
--- checking the ordering prescribed by the files UniMath/*/.packages/files ---
make: bash: Argument list too long
make: *** [Makefile:330: .check-prescribed-ordering.okay] Error 127
```
at https://github.com/UniMath/UniMath/actions/runs/7396394616/job/20121464106?pr=1817, when checking PR https://github.com/UniMath/UniMath/pull/1817.

Note: we have had the same issue with `make install` at some point: https://github.com/UniMath/UniMath/issues/1757. There, the Coq devs changed the `make` target to take a file with all the files to install.

@DanGrayson : could you take a look?
@nmvdw : fyi

**Comment by DanGrayson** [comment]
I'll take a look.

**Comment by arnoudvanderleer** [comment]
This was closed by #1826 , right?

**Comment by benediktahrens** [comment]
@arnoudvanderleer : thanks for fixing this issue!
